README.md

```diff
@@ -33,13 +33,15 @@ Install the [Firebase tools](https://firebase.google.com/docs/emulator-suite):
 curl -sL firebase.tools | bash
 ```
 
+Then copy the `./firebase-local-data.example` folder to `./firebase-local-data`. This folder will be used to store the local data of the Firebase emulator. It is ignored by git.
+
 Then start it using (replace `GOOGLE_CLOUD_PROJECT` with the value from your `.env` file):
 
 ```sh
-firebase --project GOOGLE_CLOUD_PROJECT emulators:start --import=./firebase-local-data
+firebase --project GOOGLE_CLOUD_PROJECT emulators:start --import=./firebase-local-data --export-on-exit
```

Comment:
I must admit I do not use `--project GOOGLE_CLOUD_PROJECT` on my side, I'm not sure it is required, but I may have a local default GCP project configured...